lib/redshift_ecto/connection.ex

```diff
@@ -795,7 +795,7 @@ if Code.ensure_loaded?(Postgrex) do
       do: [" DEFAULT ", to_string(literal)]
 
     defp default_expr({:ok, %{} = map}, :map) do
-      default = Ecto.Adapter.json_library().encode!(map)
+      default = json_library().encode!(map)
```

Comment:
Ecto 3 got rid of this method and instead tells you to configure `postgrex` with the json library. I went with just using the postgrex configured one for now.